src/App.tsx

```diff
@@ -16,9 +17,9 @@ export const goodsFromServer = [
 ];
 
 enum SortType {
-  NONE,
-  ALPABET,
-  LENGTH,
+  NONE = 'none',
+  ALPABET = 'alphabet',
+  LENGTH = 'length',
 }
```

Comment:
```suggestion
  NONE =,
  ALPABET,
  LENGTH,
}
```
no need to set values. you can address properties of enum like SortType.NONE / SortType.ALPABET / SortType.LENGTH